src/pages/Wallet/WalletHeader/WalletHeader.js

```diff
@@ -0,0 +1,25 @@
+import { Grid } from '@mui/material';
+import { WalletLogo, WalletPendingTransfers, WalletTitle } from './WalletHeaderStyled';
+
+const defaultLogoURL = 'https://placehold.co/192x192';
+
```

Comment:
Can we display any Material icon instead (like WalletIcon) or any image saved locally?